### ISSUE TITLE
Inline frame missing a text alternative (transcript)

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5341]
Sites identified: MCO
Page(s) identified:

- Media object page (transcript displays inside inline frame?)

WCAG compliance level: A

Description

Every inline frame (iframe) should have a text alternative that summarizes its content or purpose.

Inline frames (iframes) are used to insert content from other parts of the web.

A title that summarizes the visual content will help screen reader users to understand the iframe's purpose on the page.

See more details in [SiteImprove example](https://my2.siteimprove.com/Inspector/1167098/A11Y/Page?pageId=40886721498&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r13/failed/QlNTVHZXdlc=;NzRjZGNmNTQ=).

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbkJnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--4c63f5de06faca7c923a516fb003a5cb72f821d7/image.png)

**Comment by joncameron** [comment]
Won't be relevant past 7.7, where we'll move away from putting the transcript viewer in an iframe.

**Comment by elynema** [comment]
Since remediation is due 2/8 for MCO and IJCCR and Archivo, and 7.7 will not be rolled out until spring break at the earliest, Jon D. would like us to do these interim fixes if possible. However, this is not currently showing up as a persistent issue in SiteImprove.

However, it is still an issue at this page (https://media.dlib.indiana.edu/media_objects/zw12zv403); presumably just SiteImprove didn't hit this page on the last run.

Is there a quick way to fix this before 7.7 rolls out?

**Comment by cjcolvar** [comment]
Need to fix this in `@samvera/iiif-react-media-player` and version 1.2.2 or try upgrading to ramp 2.0 where I think this is already solved.  Probably easier to do in old codebase instead of newer ramp.

**Comment by cjcolvar** [comment]
I created https://mco-staging.dlib.indiana.edu/media_objects/2801q359p on mco-staging for testing.

**Comment by joncameron** [comment]
QA'd: I see aria-label="Attached transcript content" on the iframe that shows up when the transcript component is showing plain text.
